actions/docker_finalize_release.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: docker_finalize_release
+runner_type: orquesta
+description: Finalize docker images for st2 release
+enabled: true
+entry_point: workflows/docker_finalize_release.yaml
+parameters:
+  version:
+    type: string
+    description: Version used for the release. Should include the patch e.g. 0.1.0
+    required: true
+  fork:
+    type: string
+    description: Fork to use
+    default: StackStorm
+  host:
+    type: string
+    description: Host where to run the release process.
+  cwd:
```

Comment:
@m4dcoder Does the Orquesta runner already accept a `cwd` parameter?

Reply:
No. If I recall, the cwd param is specific to a number of release actions.